var/spack/repos/builtin/packages/cp2k/package.py

```diff
@@ -118,6 +121,12 @@ class Cp2k(MakefilePackage, CMakePackage, CudaPackage, ROCmPackage):
     variant("pytorch", default=False, description="Enable libtorch support")
     variant("quip", default=False, description="Enable quip support")
     variant("mpi_f08", default=False, description="Use MPI F08 module")
+    variant(
+        "unified_memory",
+        default=False,
+        description="Enable unified memory support on Mi250 and Mi300 GPUs",
+        when="+rocm",
```

Comment:
Is this variant also conditional on the specific `amdgpu_target`?

Reply:
good point. For now yes. I will do the necessary mods early next week